src/utils/web3-utils.js

```diff
@@ -1,9 +1,28 @@
+import { ethers, providers as Providers } from 'ethers'
 import { toChecksumAddress } from 'web3-utils'
-import env from '../environment'
-import { getDefaultChain } from '../local-settings'
+import env from '@/environment'
+import { getDefaultChain } from '@/local-settings'
 
 const DEFAULT_LOCAL_CHAIN = ''
 
+function getBackendServicesKeys() {
+  return {
+    alchemy: env('ALCHEMY_API_KEY'),
+    etherscan: env('ETHERSCAN_API_KEY'),
+    infura: env('INFURA_API_KEY'),
+    pocket: env('POCKET_API_KEY'),
+  }
+}
+
+export function getDefaultProvider() {
+  const type = getNetworkType()
```

Comment:
I think the current solution is fine. In a future version, we should provide the whole `Network` object with the specific `ENS` registry we use per network. 

This will not become a problem until we want to support AragonID.

Reply:
Good point